.github/workflows/ci.yml

```diff
@@ -63,3 +63,18 @@ jobs:
           login/base64.h login/config.h login/crypto.h login/lockdown.h \
           login/login.h login/ui.h
 
+  python_presubmit:
+    runs-on: ubuntu-18.04
+    container:
+      image: "python:3.6-buster"
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: install dependencies from PyPI
+      run: python -m pip install cryptography yapf pylint
+    - name: run unit tests
+      run: bash -c "cd python; python -m test"
```

Comment:
As far as I understand, the value of `run` should already be exceuted with `sh -c`. Can you try removing the shell wrapper?

Reply:
Fixed.